README.md

```diff
@@ -1,21 +1,36 @@
 # Welcome to DuckDuckHack!
 
-![](http://docs.duckduckhack.com/assets/hack search engine.png)
+We're a community dedicated to improving search results with open source, "Instant Answers", that solve searches in few or zero clicks.
 
-We're a community dedicated to improving search results with open source, "Instant Answers." Instant Answers use the Web's best APIs and data to solve searches in few or zero clicks. Together and openly, we can create the best search engine for every type of search.
 
-**The DuckDuckHack community and DuckDuckGo staff are working together on a mission to create the best search engine for programmers. To fully focus there, we are only accepting Pull Requests and Issues related to the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).**
+### The Programming Mission
+The current focus of DuckDuckHack is to create the best search engine for programmers. For now, we are **only accepting Pull Requests and Issues related to the Programming Mission**.
 
```

Comment:
Make `DuckDuckHack` link to duckduckhack.com here? Or is that diverting people away from the contribution path? And perhaps keep the link to the Programming Mission overview post. New people probably won't know what it is.